src/core/css/SearchResult.scss

```diff
@@ -34,10 +34,16 @@
   margin: 0;
   padding: 0;
   text-decoration: none;
+  width: calc(100% - 40px);
```

Comment:
To aid us in the future can you add comments for what the calc is based on in both cases?